vsutil/__init__.py

```diff
@@ -35,6 +38,51 @@ class Dither(Enum):
     ERROR_DIFFUSION = 'error_diffusion'  # Floyd-Steinberg error diffusion.
 
 
+def is_variable(clip: vs.VideoNode, /, *, format: bool = False, resolution: bool = False) -> bool:
```

Comment:
Not only do i feel like this logic isn't complex enough to warrant its own function, having to specify which you want to check for (as apposed to having two separate functions and making the functions even more redundant) makes it extra unwieldy.

Reply:
Agreed, if another script author wants to do something other than use the decorators with this logic it'd be easier for them to just write out the two lines `if clip.format is None; if 0 in (clip.width, clip.height)`.